OxfordBodleian/Juel-Jensen/BDLaethf30.xml

```diff
@@ -165,10 +677,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">15</dim>
                                         <dim type="left">10</dim>
                                     </dimensions>
+                                    <note> The characters per line are 10-35.</note>
+                                    <ab type="pricking">Ruling and pricking are visible. Both primary and ruling pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A/0-0/0-0/C.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling. The bottom line is written above the ruling.
+                                    </ab>
+                                    <!--double pricking f. 66, 82, 84-->
+                                    <ab type="punctuation" subtype="Dividers">One chain of red and black small semicircles after
+                                        Ps. 100 and Ps. 150, at the end of the Psalms of David, and after
+                                        the Wǝddāse Māryām for Thursday (<locus target="#76v #117v #138 #146rb"/>).
```

Comment:
Can `<ref type="work" corresp="LIT2509Weddas#Thursday"/>` be used here (like `<ref type="work" corresp="LIT2509Weddas"/>` in the layout description of p2)?